pkg/operator/ceohelpers/common.go

```diff
@@ -197,3 +200,39 @@ func VotingMemberIPListSet(ctx context.Context, cli etcdcli.EtcdClient) (sets.St
 
 	return currentVotingMemberIPListSet, nil
 }
+
+// GetBootstrapScalingStrategy determines the scaling strategy to use
+func GetBootstrapScalingStrategy(staticPodClient v1helpers.StaticPodOperatorClient, namespaceLister corev1listers.NamespaceLister, infraLister configv1listers.InfrastructureLister) (BootstrapScalingStrategy, error) {
```

Comment:
can you please move that one back to its original file? it's hard to see what changed here.

Reply:
:O ok

Reply:
moved back to bootstrap